.github/workflows/ci.yml

```diff
@@ -125,6 +125,7 @@ jobs:
         env:
           DOCKER_BUILDKIT: 1
           FML_DATABASE_URI: ${{ matrix.database-uri }}
+          FML_OUTPUT_DATABASE_URI: ${{ matrix.database-uri }}
```

Comment:
this will cause import_test to use the parameter as the destination (so we cover importing to postgres).